app/code/Magento/Email/Model/Transport.php

```diff
@@ -33,6 +35,41 @@ class Transport implements TransportInterface
      */
     const XML_PATH_SENDING_RETURN_PATH_EMAIL = 'system/smtp/return_path_email';
 
+    /**
+     * Configuration path for custom Transport
+     */
+    const XML_PATH_TRANSPORT = 'system/smtp/transport';
```

Comment:
A bit repeating with the paths.. Maybe we need a prefix and helper method?

Reply:
The configuration should probably be moved to a `Model\Config` class.